addons/html_editor/static/tests/html_field.test.js

```diff
@@ -905,6 +905,7 @@ describe("sandbox", () => {
         iframe = queryOne(
             '.o_field_html[name="txt"] iframe[sandbox="allow-same-origin allow-popups allow-popups-to-escape-sandbox"]'
         );
+        await animationFrame();
```

Comment:
we should probably use waitFor. The issue is probably link to the iframe :/

Reply:
I had a hard time trying to use waitFor, I could not manage to match elements **inside** the iframe with it.